src/components/browser-modal/browser-modal.jsx

```diff
@@ -43,34 +43,18 @@ const BrowserModal = ({intl, ...props}) => (
                     className={styles.backButton}
                     onClick={props.onBack}
                 >
-                    <FormattedMessage
-                        defaultMessage="Back"
-                        description="Label for button go back when browser is unsupported"
-                        id="gui.unsupportedBrowser.back"
-                    />
+                    Back
```

Comment:
This change is not making sense to me. In my version of `browser-modal` I have the intl and FormattedMessage parts....